hedera-node/src/main/java/com/hedera/services/state/expiry/ExpiringCreations.java

```diff
@@ -148,25 +162,43 @@ public ExpirableTxnRecord saveExpiringRecord(
 			final List<FcAssessedCustomFee> customFeesCharged,
 			final SideEffectsTracker sideEffectsTracker
 	) {
+		return createBaseRecord(memo, receiptBuilder, customFeesCharged, sideEffectsTracker, dynamicProperties.htsDefaultGasCost());
+	}
+
+	private ExpirableTxnRecord.Builder createBaseRecord(
+		final String memo,
+		final TxnReceipt.Builder receiptBuilder,
+		final List<FcAssessedCustomFee> customFeesCharged,
+		final SideEffectsTracker sideEffectsTracker,
+		final Long gasUsed
+	) {
+		// TODO: Populate log info when we have erc20 equivalence.
+		final var contractFunctionResult = ContractFunctionResult.newBuilder()
+			.setContractID(EntityIdUtils.contractParsedFromSolidityAddress(
+				Address.fromHexString(HTS_PRECOMPILED_CONTRACT_ADDRESS)))
+			.setGasUsed(gasUsed);
 		if (sideEffectsTracker.hasTrackedNewTokenId()) {
 			receiptBuilder.setTokenId(EntityId.fromGrpcTokenId(sideEffectsTracker.getTrackedNewTokenId()));
 		}
 		if (sideEffectsTracker.hasTrackedTokenSupply()) {
 			receiptBuilder.setNewTotalSupply(sideEffectsTracker.getTrackedTokenSupply());
+			contractFunctionResult.setContractCallResult(ByteString.copyFrom(SerializationUtils.serialize(sideEffectsTracker.getTrackedTokenSupply())));
 		}
 		if (sideEffectsTracker.hasTrackedNftMints()) {
 			final var serialNoList = sideEffectsTracker.getTrackedNftMints();
 			final var rawSerials = new long[serialNoList.size()];
 			Arrays.setAll(rawSerials, serialNoList::get);
 			receiptBuilder.setSerialNumbers(rawSerials);
+			contractFunctionResult.setContractCallResult(ByteString.copyFrom(SerializationUtils.serialize(sideEffectsTracker.getTrackedNftMints().toArray())));
```

Comment:
I'm not sure this is correct, the values I observe in child records looks like this:
```
  contractCallResult: "\254\355\000\005ur\000\023[Ljava.lang.Object;\220\316X\237\020s)l\002\000\000xp\000\000\000\001sr\000\016java.lang.Long;\213\344\220\314\217#\337\002\000\001J\000\005valuexr\000\020java.lang.Number\206\254\225\035\v\224\340\213\002\000\000xp\000\000\000\000\000\000\000\001"
```
We shouldn't have java objects here, rather the Bytes that are returned to the solidity contract by the precompile execution.